operations/services.md

```diff
@@ -931,3 +931,75 @@ Creates a new order with the specified products and items. Only positive charges
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CustomerIds": [
+        "35d4b117-4e60-44a3-9580-c582117eff98"
+    ],
+    "ReservationIds": [
+        "bfee2c44-1f84-4326-a862-5289598f6e2d"
+    ],
+    "ReservationGroupIds": [
+        "c704dff3-7811-4af7-a3a0-7b2b0635ac59"
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CustomerIds` | array of string | optional | Unique identifiers of [Customer](customers.md#customer)s. |
+| `ReservationIds` | array of string | optional | Unique identifiers of [Reservation](reservations.md#reservation)s. |
+| `ReservationGroupIds` | array of string | optional | Unique identifiers of [Reservation group](reservations.md#reservation-group)s. |
+| `Extent` | [Companionship extent](services.md#companionship-extent) | optional | Extent of data to be returned. E.g. it is possible to specify that together with the companionships, customers, reservations, and reservation groups should be also returned. If not specified, only `Companionships` are returned. |
+
+#### Companionship extent
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Customers` | bool | optional | Whether the response should contain customers. |
+| `Reservations` | bool | optional | Whether the response should contain reservations. |
+| `ReservationGroups` | bool | optional | Whether the response should contain reservation groups. |
+
+### Response
+
+```javascript
+{
+    "Companionships": [
+        {
+            "CustomerId": "35d4b117-4e60-44a3-9580-c582117eff98",
+            "ReservationGroupId": "c704dff3-7811-4af7-a3a0-7b2b0635ac59",
+            "ReservationId": "bfee2c44-1f84-4326-a862-5289598f6e2d"
+        }
+    ],
+    "Customers": null,
+    "Reservations": null,
+    "ReservationGroups": null
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Companionships` | array of [Companionship](services.md#companionship) | required | Companionships. |
+| `Customers` | array of [Customer](customers.md#customer) | optional | Customers that belongs to companionship. |
+| `Reservations` | array of [Reservation](reservations.md#reservation) | optional | The reservations that belongs to companionship. |
+| `ReservationGroups` | array of [Reservation group](reservations.md#reservation-group) | optional | Reservation groups that belongs to companionship. |
+
+#### Companionship
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `CustomerId` | string | required | Unique identifier of [Customer](customers.md#customer). |
+| `ReservationId` | string | required | Unique identifier of [Reservation](reservations.md#reservation). |
```

Comment:
This is optional, no?

Reply:
I'm not sure what is the difference between required and optional in response - this will be always returned so I made it required.

Reply:
This is really optional. You can have companionship without the assignment to reservation. Showed in person.